src/corecel/Assert.hh

```diff
@@ -476,6 +478,13 @@ class RuntimeError : public std::runtime_error
     //! Access detailed information
     RuntimeErrorDetails const& details() const { return details_; }
 
+    //!@{
+    //! String constants for "which" error message
+    static char const validate_err_str[];
+    static char const not_config_err_str[];
+    static char const not_impl_err_str[];
```

Comment:
What about this
```suggestion
    static constexpr char validate_err_str[] {"runtime"};
    static constexpr char not_config_err_str[] {"configuration"};
    static constexpr char not_impl_err_str[] {"implementation"};
```

or using `string_view`?

Reply:
That would require making those constants `inline` and making the linker do the work. I think it's fine to keep the definition of those constants in the `.cc` file where they could be changed without forcing the whole codebase to recompile.

What's the advantage of using string_view? The `RuntimeError.which` now takes a `char const*` C string so a string_view would technically conflict with that.

Reply:
No big advantage for string_view here, it just provides more of a c++ interface at a cost of a few extra bytes.